### ISSUE TITLE
Error when auth-provider is oidc

### ISSUE THREAD

**Issue by ulrichSchreiner** [groundcover-com/murre#11]
hi,

when trying to use `murre` with a KUBECONFIG where the user has an `auth-provider`  with name `oidc` i get

```
❯ murre --kubeconfig $KUBECONFIG
panic: no Auth Provider found for name "oidc"

goroutine 1 [running]:
main.main()
        /home/usc/go/pkg/mod/github.com/groundcover-com/murre@v0.0.2/main.go:32 +0x112
```

my KUBECONFIG looks like this:

```yaml
users:
  - name: userid
    user:
      auth-provider:
        config:
          client-id: XXXX
          client-secret: XXXX
          id-token: XXXX
          idp-certificate-authority: XXXX
          idp-issuer-url: XXXX
          refresh-token: XXXX
        name: oidc
```

**Comment by maxlevinps** [comment]
@ulrichSchreiner - We've released v0.0.3 which should resolve the missing oidc provider. It will be great if you'll be able to test it and report wether the fix resolves your issue.

**Comment by ulrichSchreiner** [comment]
yes it works, thanks
